doc/stdlib/punctuation.rst

```diff
@@ -13,7 +13,7 @@ symbol      meaning
 ``#``       begin single line comment
 ``#=``      begin multi-line comment (these are nestable)
 ``=#``      end multi-line comment
-``$``       bitwise xor operator, string and expression interpolation
+``⊻``       bitwise xor operator, string and expression interpolation
```

Comment:
The "string and expression interpolation" does not apply to `⊻`, `$` should be kept for that.